Lib/test/test_compileall.py

```diff
@@ -460,31 +460,18 @@ def test_error(self):
 class CommandLineTestsBase:
     """Test compileall's CLI."""
```

Comment:
When I reviewed this PR, I failed to find where `self.directory` comes from. Usually, the setUp() method is defined first in a test case. Would you move to move the setUp() of this class at the beginning? It would help to see where self.directory comes from.

Reply:
Sure.  The test suite as a whole is pretty much a free-for-all style-wise.